File: algo/problems/0242-valid-anagram/js/validAnagram.test.js
const { isAnagram } = require('./validAnagram');

test('true', () => { expect(isAnagram('anagram', 'nagaram')).toBe(true); });
test('false', () => { expect(isAnagram('rat', 'car')).toBe(false); });
test('diff len', () => { expect(isAnagram('a', 'ab')).toBe(false); });

File: algo/problems/0242-valid-anagram/scala/ValidAnagramTest.scala
//> using scala 3.4.2
//> using test.dep org.scalameta::munit::1.0.0

class ValidAnagramTest extends munit.FunSuite:
  test("valid")    { assert(ValidAnagram.isAnagram("anagram", "nagaram")) }
  test("invalid")  { assert(!ValidAnagram.isAnagram("rat", "car")) }
  test("diff len") { assert(!ValidAnagram.isAnagram("a", "ab")) }
